ci/spatial.bat

```diff
@@ -0,0 +1 @@
+bash "%~dp0spatial" %*
```

Comment:
Why is this needed?

Reply:
We have CMake rules that call `spatial` which means it must be on the PATH. On Windows the `ci/spatial` bash script isn't picked up as an executable. So we introduce this batch script which will be run on Windows when running `spatial`. It just forwards to the bash script.